Assets/MRTK/SDK/Features/UX/Scripts/PressableButtons/PressableButton.cs

```diff
@@ -375,7 +375,13 @@ private void PulseProximityLight()
             {
                 foreach (var pointer in currentInputSources[currentInputSources.Count - 1].Pointers)
                 {
-                    ProximityLight[] proximityLights = pointer.BaseCursor?.GameObjectReference?.GetComponentsInChildren<ProximityLight>();
+                    GameObject cursorGameObject = pointer.BaseCursor?.GameObjectReference;
```

Comment:
Do we need a separate null check for base cursor as I think it is also MonoBehaviour?

Reply:
ahh good catch! I was thrown off since the property refers to the interface, but you're right that most/all consumers of that interface are MonoBehaviours

Reply:
Ah I did not realize it is referencing the interface... In that case a separate null check will not properly check Unity null either so I think we can leave it as is.

Reply:
We have some custom methods for checking all null paths on types we expect can be Unity objects, so I updated this and cleaned up a few other spots